### ISSUE TITLE
[FEATURE]: Add Hindi captions to `Images.JSON`

### ISSUE THREAD

**Issue by adityaraute** [rbjks/rbjks.github.io#103]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Currently we only have English text in the `images.json` file for all activities.

### Proposed Solution

Added a Hindi caption to each image which can be later used for translations.

### Do you want to work on this issue?

No

**Comment by Chanchal-D** [comment]
Yes, I would like to contribute to this feature and that can be done by first, understanding its structure—an array of image objects. Then, introduce a new field, `"hindi_caption"`, to accommodate Hindi captions. Populate this field for each image object with corresponding captions. Validate the JSON for correctness and test to ensure the Hindi captions display properly.

**Comment by adityaraute** [comment]
Sounds perfect @Chanchal-D !

**Comment by adityaraute** [comment]
Hi @Chanchal-D 
Please provide an update on this issue

**Comment by adityaraute** [comment]
Hi @Chanchal-D 

Please update on this issue by June 10. 
We will have to unassign you if no update is received by then.

**Comment by kripa-sindhu-007** [comment]
@adityaraute I guess it's 3 days passed June 10 now :)

**Comment by adityaraute** [comment]
Thanks for informing. 

@Chanchal-D unassigning you from this issue due to lack of response.

**Comment by kripa-sindhu-007** [comment]
@adityaraute can you assign me this issue.i sent a pull request already for this 😅

**Comment by adityaraute** [comment]
Done. Please link the Pull Request to the issue in the Development section on the right hand side.

**Comment by kripa-sindhu-007** [comment]
I can do it on Tuesday. I came to my village without my laptop 🥲.

**Comment by adityaraute** [comment]
Hi @kripa-sindhu-007 please share an update on the issue
